Tests/UnitTests/SubscriberAttributes/PurchasesSubscriberAttributesTests.swift

```diff
@@ -794,7 +794,9 @@ class PurchasesSubscriberAttributesTests: TestCase {
         let token = "token"
 
         self.mockAttributionFetcher.adServicesTokenToReturn = token
```

Comment:
this test worked even if you comment out the purchasing part, because ad services tokens get posted right away, once per install. This removes that behavior so we can actually test what happens in post receipt